server_addon/houdini/client/ayon_houdini/plugins/publish/validate_usd_render_arnold.py

```diff
@@ -35,7 +35,11 @@ def process(self, instance):
         )
         render_chunk_size = submission_data.get("chunk", 1)
         export_chunk_size = submission_data.get("export_chunk", 1)
-        usd_file_per_frame = "$F" in instance.data["ifdFile"]
+        usd_file_per_frame = (
+            instance.data["creator_attributes"].get("render_target") == "farm_split"
```

Comment:
Could you elaborate once more why this particular check is important? Still feels off that this influences whether or not it considers the export as usd file per frame or not?

Reply:
Ah I think this check is included because of the issue [you mentioned here](https://github.com/BigRoy/ayon-core/pull/8#discussion_r1627802568):

> So, in the current implementation "ifdFile" exists only if the render target was set to "farm_split" aka "Farm Rendering - Split export & render jobs".
> So, this validator will crash for other render targets. because of a key error.

Reply:
I'll merge and resolve things in my branch instead.

Reply:
Tweaked with https://github.com/ynput/ayon-core/pull/295/commits/7d35eee3ed5bb0bc23bba7ddd87ad28a600b0861